gem/optimise.py

```diff
@@ -493,3 +589,7 @@ def aggressive_unroll(expression):
     expression, = unroll_indexsum((expression,), predicate=lambda index: True)
     expression, = remove_componenttensors((expression,))
     return expression
+
+
+def fast_sum_factorise(sum_indices, factors):
+    return sum_factorise(*delta_elimination(list(reversed(sum_indices)), factors))
```

Comment:
Why is this called `fast_sum_factorise`? What is the difference between this and `sum_factorise`?